framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/resource/impl/ProjectImpl.java

```diff
@@ -162,6 +163,42 @@ public void createResources(String resourceUrl) {
         }
     }
 
+    public void createResourcesFromYaml(String yamlUrl) {
+        final OpenShiftBinary oc = OpenShifts.masterBinary(this.getName());
```

Comment:
The code in these methods is duplicated. Please separate it in dedicated method(s).